### PR TITLE
UEFI boot

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: configs/config.clevo_ns70pu [hunk 1]
```diff
@@ -9,6 +9,8 @@ CONFIG_CBFS_SIZE=0xA00000
 CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NS5x_NS7xPU"
 # CONFIG_USE_PM_ACPI_TIMER is not set
 CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_EDK2_BOOT_TIMEOUT=4
+CONFIG_USE_LEGACY_8254_TIMER=y
```
Comment (mkopec) [inline configs/config.clevo_ns70pu:13]: Unneeded, causes sleep issues and increased power draw. Please remove

Comment (Stojek139808) [reply configs/config.clevo_ns70pu]: Removed.

File: configs/config.clevo_ns70pu [hunk 2]
```diff
@@ -9,6 +9,8 @@ CONFIG_CBFS_SIZE=0xA00000
 CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NS5x_NS7xPU"
 # CONFIG_USE_PM_ACPI_TIMER is not set
 CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_EDK2_BOOT_TIMEOUT=4
```
Comment (mkopec) [inline configs/config.clevo_ns70pu:12]: Shouldn't this be 2 on NovaCustom laptops?

Comment (Stojek139808) [reply configs/config.clevo_ns70pu]: Doesn't change that much but changed it to 2, now it boots a bit faster since it waits 2s less.

**Pull request merged**